examples/playground/package.json

```
@@ -8,13 +8,14 @@
   "dependencies": {
     "@openfun/verna": "../../src",
     "react": "17.0.2",
-    "react-dom": "17.0.2"
+    "react-dom": "17.0.2",
```

This is not valid in JSON files.
```suggestion
    "react-dom": "17.0.2"
```

---

Screwed up my rebase, my bad